cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,28 @@
 /// <reference types='cypress' />
-
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'qa_jan24',
+    password: '12345Qwert!'
+  };
 
+  it('should Log in a user', () => {
+    cy.visit('/login');
+    cy.get('#userName').type(user.username);
+    cy.get('#password').type(user.password);
+    cy.get('#login').click();
+    cy.url().should('not.include', '/login');
+    cy.get('#userName-value').should('contain', user.username);
+    cy.url().should('include', '/profile');
   });
 
-  it('', () => {
-
+  it('should search the book', () => {
+    cy.login();
+    cy.visit('/profile');
+    cy.contains('#item-2', 'Book Store').click();
+    cy.get('#searchBox').type(`Speaking JavaScript{enter}`);
```

Comment:
```suggestion
    cy.get('#searchBox').type(`Speaking JavaScript{enter}`);
```
create a const for 'Speaking JavaScript' string, you use it two times